applications/FreeSurfaceApplication/python_scripts/free_surface_analysis.py

```diff
@@ -24,10 +24,24 @@ def Initialize(self):
 
         # Initialize remaining nodes
         density = self.project_parameters["solver_settings"]["density"].GetDouble()
-        body_force = self.project_parameters["solver_settings"]["body_force"].GetVector()
         small_value = 1e-4
         active_node_count = 0
+
+        # Initialize body force value
+        body_force = KratosMultiphysics.Vector(3)
+        body_force[0] = 0.0
+        body_force[1] = 0.0
+        body_force[2] = 0.0
```

Comment:
```suggestion
```
I think you can remove these as you're overwriting all the components below.

Reply:
Done